.github/workflows/renv.yml

```diff
@@ -0,0 +1,138 @@
+---
+name: Propagate renv.lock
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+    branches:
+      - main
+      - devel
+      - pre-release
+
+# TODO find and replace:
+# 'renv.lock.test' -> 'renv.lock'
+
+jobs:
+  validate:
+    name: Validate renv.lock
+    runs-on: ubuntu-20.04
+    container:
+      image: ghcr.io/insightsengineering/rstudio_4.2.1_bioc_3.15:latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Validate renv.lock
+        run: |
+          renv::init()
```

Comment:
Maybe you don't need to `init`?

Reply:
Confirmed - this doesn't seem necessary.